.circleci/config.yml

```diff
@@ -253,6 +253,8 @@ jobs:
                 --enable_variadic_signatures \
                 --velox_fuzzer_enable_complex_types \
                 --lazy_vector_generation_ratio 0.2 \
+                --velox_fuzzer_enable_column_reuse \
+                --velox_fuzzer_enable_expression_reuse \
```

Comment:
Are we going to enable the new features right away? Shall we do it in a separate PR after ensuring there is no fuzzer test failure?

Reply:
sure, we can do it in a separate PR, will remove it